internal/command/proxy/proxy.go

```diff
@@ -19,8 +19,8 @@ import (
 
 func New() *cobra.Command {
 	var (
-		long  = strings.Trim(`Proxies connections to a fly VM through a Wireguard tunnel The current application DNS is the default remote host`, "\n")
-		short = `Proxies connections to a fly VM`
+		long  = strings.Trim(`Proxies connections to a Fly Machine through a WireGuard tunnel. The current application DNS is the default remote host.`, "\n")
```

Comment:
"By default, connects to the first Machine address returned by an internal DNS query on the app"?